Simulation.jl

```diff
@@ -0,0 +1,50 @@
+function simulate_deposition(fcoeffs, Ea_values, T_range, num_steps, dt)
```

Comment:
add a docstring that explains the function and what the inputs/outputs are

Reply:
also, I would suggest just writing this for a single T value. It should work fine to compute a range either by a custom broadcast or a list comprehension, but I think for the sake of modularity, just having the one loop over `num_steps` inside this function makes more sense

Reply:
(likewise for a single `fcoeff` vector rather than a vector of vectors)